runtime/server/column_api.go

```diff
@@ -352,7 +347,12 @@ func (s *Server) GetNumericHistogram(ctx context.Context, request *api.NumericHi
 	histogramBins := make([]*api.NumericHistogramBins_Bin, 0)
 	for histogramRows.Next() {
 		bin := &api.NumericHistogramBins_Bin{}
-		rows.Scan(&bin.Bucket, &bin.Low, &bin.High, &bin.Count)
+		rowMap := make(map[string]any)
+		histogramRows.MapScan(rowMap)
+		bin.Bucket = rowMap["bucket"].(int64)
+		bin.Low = rowMap["low"].(float64)
+		bin.High = rowMap["high"].(float64)
+		bin.Count = rowMap["count"].(*big.Int).Int64()
```

Comment:
These type casts make me a little nervous as they'll panic if the type doesn't match perfectly. If it's just the `count` that can't automatically be coerced, consider doing:

```go
var count *big.Int
rows.Scan(&bin.Bucket, &bin.Low, &bin.High, &count)
bin.Count = count.Int64()
```

Also, I think `rows.Scan` can return an error which we should handle

Reply:
its not just big int, somehow scanning gives all empty values for other fields as well.
```
		var count = new(big.Int)
		rows.Scan(&bin.Bucket, &bin.Low, &bin.High, &count)
		bin.Count = count.Int64()
```
this code initializes bucket, low and high with empty values

Reply:
Sounds like this needs more debugging then. Try to catch the error from `Scan`, maybe it's instructive.

The potential problem around using a map is that `MapScan` will store the values as the type pointed out by the DB driver. If they change that to e.g. `int` instead of `int64`, then suddenly the type assertions here will break. On the other hand, `Scan` tries to cast/coerce the values from the DB driver into the types you provide it, and only fails if there is no way to convert.

Reply:
yes there is an error, it says `sql: Rows are closed`

Reply:
I see one issue, used wrong variable here

Reply:
fixed